packages/app-schema-router/README.md

```diff
@@ -29,10 +29,10 @@ module.exports = {
     new GraphQLAppPlayground({ apiPath })
     new SchemaRouterApp({
       apiPath,
-      routerFn: (req) => req.session.keystoneItemId ? 'private' : 'public',
+      routerFn: (req) => req.session.keystoneItemId ? 'private' : 'admin',
```

Comment:
As `private` is not a reserved name anymore, can we bring back the `private` now, and revert this ternary condition?